<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultisigDialog</class>
 <widget class="QDialog" name="MultisigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2117</width>
    <height>1125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multisig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Create Multisig</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Keys required:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxCreateMSReq">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNReq">
           <property name="text">
            <string>of 0 total key(s) required to transfer from multisig</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEditCreateMS">
           <property name="font">
            <font>
             <family>Noto Mono</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Enter pubkeys one per line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowserCreateMSOutput">
           <property name="font">
            <font>
             <family>Noto Mono</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Start entering keys to begin!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Start Multisig Transfer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>TxID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTransferTXID"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Amount In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BitcoinAmountField" name="amountTransferIn"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Output n</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxTransferN"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Destination (One only for now..)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTransferDest"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Change Destination </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTransferDestChange"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BitcoinAmountField" name="amountTransfer"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BitcoinAmountField" name="amountTransferFee"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTransfer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Start Transfer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Unsigned multisig transfer transaction:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowserTransferOutput">
         <property name="font">
          <font>
           <family>Noto Mono</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Sign Multisig</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Prevout amount</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BitcoinAmountField" name="amountSign"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>P2SH Script Hex:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSignP2SH"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Redeem Script Hex:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSignRedeem"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Transaction Hex:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSignTx"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Private Key Hex:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSignKey"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSign">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Sign</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="textBrowserSignOutput"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/drivechainamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
